File: codes/202606/16/DataModel.cs
// DataModel.cs
// CSV 파일의 각 행을 나타내는 데이터 모델 클래스입니다.

using System;

namespace CsvDataProcessor
{
    public class DataModel
    {
        // 데이터의 고유 식별자 또는 이름입니다.
        public string Name { get; set; }

        // 데이터의 카테고리입니다.
        public string Category { get; set; }

        // 데이터와 관련된 숫자 값입니다.
        public int Value { get; set; }

        // DataModel 객체를 문자열로 표현하는 메서드입니다.
        public override string ToString()
        {
            return $"이름: {Name}, 카테고리: {Category}, 값: {Value}";
        }
    }
}

File: codes/202606/29/src/main/java/DataModel.java
// DataModel.java
/**
 * CSV 파일의 각 행을 나타내는 데이터 모델 클래스입니다.
 * 이 클래스는 이름, 나이, 점수 정보를 저장합니다.
 */
public class DataModel {
    private String name;
    private int age;
    private double score;

    /**
     * DataModel의 생성자입니다.
     * @param name 이름
     * @param age 나이
     * @param score 점수
     */
    public DataModel(String name, int age, double score) {
        this.name = name;
        this.age = age;
        this.score = score;
    }

    // Getter 메서드
    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    public double getScore() {
        return score;
    }

    /**
     * 객체의 문자열 표현을 반환합니다.
     * @return 객체 정보가 포함된 문자열
     */
    @Override
    public String toString() {
        return "DataModel{" +
               "name='" + name + ''' +
               ", age=" + age +
               ", score=" + score +
               '}';
    }
}
